README.md

```diff
@@ -1 +1,26 @@
 # git-diffie
+
+## got nix?
+
+```sh
+$ nix run github:the6p4c/git-diffie\#git-diffie
+```
+
+## got nixos?
+
+```nix
+{
+  inputs = {
+    your-nixos = "github:NixOS/nixpkgs/nixos-???";
+    git-diffie.url = "github:the6p4c/git-diffie";
+    git-diffie.inputs.nixpkgs.follows = "your-nixos";
+  };
+  outputs = inputs@{ git-diffie, ... }: {
+    nixosConfigurations.machine = ... {
+      modules = [
+        { environment.systemPackages = [git-diffie.packages.x86_64-linux.default]; }
```

Comment:
@Sorixelle there’s got to be a better way™

Reply:
Since you made an overlay (smartuppy!):

```nix
modules = [
  ({ pkgs, ... }: {
    nixpkgs.overlays = [ git-diffie.overlays.default ];
    environment.systemPackages = [ pkgs.git-diffie ];
  })
];
```

Reply:
ty!! done in 5cd7eac0567ecdf0d46be62110b6fd88b40af5d2